src/_zkapauthorizer/controller.py

```diff
@@ -195,6 +195,9 @@ class IndexedRedeemer(object):
 
     redeemers = attr.ib()
 
+    def tokens_to_passes(self, message, unblinded_tokens):
+        raise NotImplementedError("IndexedRedeemer cannot create passes")
```

Comment:
This method is part of the interface and mypy doesn't like incomplete implementations.